e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-trading-address-error.spec.js

```diff
@@ -0,0 +1,76 @@
+import { companyDetails } from '../../../../pages/your-business';
+import { submitButton, yesRadioInput, inlineErrorMessage } from '../../../../pages/shared';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import partials from '../../../../partials';
+import { ROUTES, FIELD_IDS, COMPANIES_HOUSE_NUMBER } from '../../../../../../constants';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const {
+  EXPORTER_BUSINESS: {
+    YOUR_COMPANY: {
+      TRADING_ADDRESS,
+    },
+  },
+} = FIELD_IDS.INSURANCE;
+
+const COMPANY_DETAILS_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
+
+describe("Your business - As an Exporter I want to enter details about my business in 'your business' section - trading address errors", () => {
```

Comment:
We should change the field name to be `errors.spec.js` or `-validation.spec` like the others

Reply:
done